File: cite_culture/src/Entities/Evenement.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entities;

import java.util.Date;

/**
 *
 * @author mouna dridi
 */
public class Evenement {
    protected int idEvent;
    protected String titre;
    protected String description;
    protected Date dateD;
    protected Date dateF;
    protected String salle;
    protected float prixEnfant;
    protected float prixAdulte;
    protected float prixEtudiant;
    protected String time1;
    protected String typeEvent;
    protected String image;
    protected int nbrPlace;

    public Evenement() {
    }

    
    public Evenement(String titre, String description, Date dateD, Date dateF, String salle, float prixEnfant, float prixAdulte, float prixEtudiant, String time1, String typeEvent, String image, int nbrPlace) {
        this.titre = titre;
        this.description = description;
        this.dateD = dateD;
        this.dateF = dateF;
        this.salle = salle;
        this.prixEnfant = prixEnfant;
        this.prixAdulte = prixAdulte;
        this.prixEtudiant = prixEtudiant;
        this.time1 = time1;
        this.typeEvent = typeEvent;
        this.image = image;
        this.nbrPlace = nbrPlace;
    }

    public int getIdEvent() {
        return idEvent;
    }

    public String getTitre() {
        return titre;
    }

    public String getDescription() {
        return description;
    }

    public Date getDateD() {
        return dateD;
    }

    public Date getDateF() {
        return dateF;
    }

    public String getSalle() {
        return salle;
    }

    public float getPrixEnfant() {
        return prixEnfant;
    }

    public float getPrixAdulte() {
        return prixAdulte;
    }

    public float getPrixEtudiant() {
        return prixEtudiant;
    }

    public String getTime1() {
        return time1;
    }

    public String getTypeEvent() {
        return typeEvent;
    }

    public String getImage() {
        return image;
    }

    public int getNbrPlace() {
        return nbrPlace;
    }

    public void setIdEvent(int idEvent) {
        this.idEvent = idEvent;
    }

    public void setTitre(String titre) {
        this.titre = titre;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public void setDateD(Date dateD) {
        this.dateD = dateD;
    }

    public void setDateF(Date dateF) {
        this.dateF = dateF;
    }

    public void setSalle(String salle) {
        this.salle = salle;
    }

    public void setPrixEnfant(float prixEnfant) {
        this.prixEnfant = prixEnfant;
    }

    public void setPrixAdulte(float prixAdulte) {
        this.prixAdulte = prixAdulte;
    }

    public void setPrixEtudiant(float prixEtudiant) {
        this.prixEtudiant = prixEtudiant;
    }

    public void setTime1(String time1) {
        this.time1 = time1;
    }

    public void setTypeEvent(String typeEvent) {
        this.typeEvent = typeEvent;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public void setNbrPlace(int nbrPlace) {
        this.nbrPlace = nbrPlace;
    }

    @Override
    public String toString() {
        return "Evenement{" + "idEvent=" + idEvent + ", titre=" + titre + ", description=" + description + ", dateD=" + dateD + ", dateF=" + dateF + ", salle=" + salle + ", prixEnfant=" + prixEnfant + ", prixAdulte=" + prixAdulte + ", prixEtudiant=" + prixEtudiant + ", time1=" + time1 + ", typeEvent=" + typeEvent + ", image=" + image + ", nbrPlace=" + nbrPlace + '}';
    }
    
}

File: src/EventBundle/Entity/Evenement.php
<?php

namespace EventBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Evenement
 *
 * @ORM\Table(name="evenement")
 * @ORM\Entity
 */
class Evenement
{
    /**
     * @var integer
     *
     * @ORM\Column(name="idEvent", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idevent;

    /**
     * @var string
     *
     * @ORM\Column(name="titre", type="string", length=255, nullable=false)
     */
    private $titre;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="string", length=255, nullable=false)
     */
    private $description;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="dateD", type="date", nullable=false)
     */
    private $dated;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="dateF", type="date", nullable=false)
     */
    private $datef;

    /**
     * @var string
     *
     * @ORM\Column(name="salle", type="string", length=10, nullable=false)
     */
    private $salle;

    /**
     * @var float
     *
     * @ORM\Column(name="prixEnfant", type="float", precision=10, scale=0, nullable=false)
     */
    private $prixenfant;

    /**
     * @var float
     *
     * @ORM\Column(name="prixAdulte", type="float", precision=10, scale=0, nullable=false)
     */
    private $prixadulte;

    /**
     * @var float
     *
     * @ORM\Column(name="prixEtudiant", type="float", precision=10, scale=0, nullable=false)
     */
    private $prixetudiant;

    /**
     * @var string
     *
     * @ORM\Column(name="time", type="string", length=30, nullable=false)
     */
    private $time;

    /**
     * @var string
     *
     * @ORM\Column(name="typeEvent", type="string", length=30, nullable=false)
     */
    private $typeevent;

    /**
     * @var string
     *
     * @ORM\Column(name="image", type="string", length=255, nullable=false)
     */
    private $image;

    /**
     * @var integer
     *
     * @ORM\Column(name="nbrPlace", type="integer", nullable=false)
     */
    private $nbrplace;



    /**
     * Get idevent
     *
     * @return integer
     */
    public function getIdevent()
    {
        return $this->idevent;
    }

    /**
     * Set titre
     *
     * @param string $titre
     *
     * @return Evenement
     */
    public function setTitre($titre)
    {
        $this->titre = $titre;

        return $this;
    }

    /**
     * Get titre
     *
     * @return string
     */
    public function getTitre()
    {
        return $this->titre;
    }

    /**
     * Set description
     *
     * @param string $description
     *
     * @return Evenement
     */
    public function setDescription($description)
    {
        $this->description = $description;

        return $this;
    }

    /**
     * Get description
     *
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * Set dated
     *
     * @param \DateTime $dated
     *
     * @return Evenement
     */
    public function setDated($dated)
    {
        $this->dated = $dated;

        return $this;
    }

    /**
     * Get dated
     *
     * @return \DateTime
     */
    public function getDated()
    {
        return $this->dated;
    }

    /**
     * Set datef
     *
     * @param \DateTime $datef
     *
     * @return Evenement
     */
    public function setDatef($datef)
    {
        $this->datef = $datef;

        return $this;
    }

    /**
     * Get datef
     *
     * @return \DateTime
     */
    public function getDatef()
    {
        return $this->datef;
    }

    /**
     * Set salle
     *
     * @param string $salle
     *
     * @return Evenement
     */
    public function setSalle($salle)
    {
        $this->salle = $salle;

        return $this;
    }

    /**
     * Get salle
     *
     * @return string
     */
    public function getSalle()
    {
        return $this->salle;
    }

    /**
     * Set prixenfant
     *
     * @param float $prixenfant
     *
     * @return Evenement
     */
    public function setPrixenfant($prixenfant)
    {
        $this->prixenfant = $prixenfant;

        return $this;
    }

    /**
     * Get prixenfant
     *
     * @return float
     */
    public function getPrixenfant()
    {
        return $this->prixenfant;
    }

    /**
     * Set prixadulte
     *
     * @param float $prixadulte
     *
     * @return Evenement
     */
    public function setPrixadulte($prixadulte)
    {
        $this->prixadulte = $prixadulte;

        return $this;
    }

    /**
     * Get prixadulte
     *
     * @return float
     */
    public function getPrixadulte()
    {
        return $this->prixadulte;
    }

    /**
     * Set prixetudiant
     *
     * @param float $prixetudiant
     *
     * @return Evenement
     */
    public function setPrixetudiant($prixetudiant)
    {
        $this->prixetudiant = $prixetudiant;

        return $this;
    }

    /**
     * Get prixetudiant
     *
     * @return float
     */
    public function getPrixetudiant()
    {
        return $this->prixetudiant;
    }

    /**
     * Set time
     *
     * @param string $time
     *
     * @return Evenement
     */
    public function setTime($time)
    {
        $this->time = $time;

        return $this;
    }

    /**
     * Get time
     *
     * @return string
     */
    public function getTime()
    {
        return $this->time;
    }

    /**
     * Set typeevent
     *
     * @param string $typeevent
     *
     * @return Evenement
     */
    public function setTypeevent($typeevent)
    {
        $this->typeevent = $typeevent;

        return $this;
    }

    /**
     * Get typeevent
     *
     * @return string
     */
    public function getTypeevent()
    {
        return $this->typeevent;
    }

    /**
     * Set image
     *
     * @param string $image
     *
     * @return Evenement
     */
    public function setImage($image)
    {
        $this->image = $image;

        return $this;
    }

    /**
     * Get image
     *
     * @return string
     */
    public function getImage()
    {
        return $this->image;
    }

    /**
     * Set nbrplace
     *
     * @param integer $nbrplace
     *
     * @return Evenement
     */
    public function setNbrplace($nbrplace)
    {
        $this->nbrplace = $nbrplace;

        return $this;
    }

    /**
     * Get nbrplace
     *
     * @return integer
     */
    public function getNbrplace()
    {
        return $this->nbrplace;
    }
}
